README.md

```diff
@@ -10,7 +10,7 @@ the rights to use your contribution. For details, visit https://cla.microsoft.co
 
 To contibute a connector to the open source repo, please start by creating a fork on the github repo.
 Once you have the fork created, create a new branch on the forked repo. Clone this forked repo on you 
-local machine, and checkout the branch. Create a folder for your connector under the `connectors` folder
+local machine, and checkout the branch. Create a folder for your connector under the `certified-connectors` folder
```

Comment:
We should probably include a distinction here.

Certified connectors directed to open source by Microsoft should be under certified-connectors
All other connectors should be under custom-connectors.

Reply:
@bezhan-msft fixed in latest commit